lighty-modules/lighty-gnmi/lighty-gnmi-sb/src/main/java/io/lighty/gnmi/southbound/schema/certstore/rpc/CertificationStorageServiceRpcImpl.java

```diff
@@ -80,4 +89,11 @@ public void onFailure(final Throwable throwable) {
         }, MoreExecutors.directExecutor());
         return rpcResult;
     }
+
+    public ClassToInstanceMap<Rpc<?,?>> getRpcClassToInstanceMap() {
```

Comment:
IMO this can be simplified by new logic.

Reply:
https://github.com/PANTHEONtech/lighty/commit/8b87c414b754954a0e04733fb92e815556bfecae

Reply:
OK, there are some other occurrences of this - but its not that important to migrate - the usage is simple enough.